tests/test_basics.py

```diff
@@ -156,7 +156,8 @@ def test_multi_get_opt_ok(self):
         (rc, get_kvs) = yield self.c.multi_get_opt(self.TEST_HKEY,
                                                    self.TEST_SKEY + rand_key + '_0',
                                                    self.TEST_SKEY + rand_key + '_' + '9'*len(str(count)),
-                                                   opt)
+                                                   opt,
+                                                   500)
```

Comment:
Why did you fix these tests?

Reply:
Because it can't pass while I start 2.0.0 version onebox. For `multi_get`, 2.0.0 use count as iteration_count, sometimes, the unit test may be failed.